### PR TITLE
Make bottom 3 teams red regardless of league size

### PR DESCRIPTION
**Pull request by thurask**
As it stands, teams 18-20 are marked in red; if the league doesn't have 20 teams (ex. Bundesliga), that's a problem.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -105,14 +105,14 @@ def print_standings(league_table):
 					position="POS", team="CLUB", played="PLAYED", 
 					goaldiff="GOAL DIFF", points="POINTS"
 			))
-
+	positionlist = (team["position"] for team in league_table["standing"])
```
Comment (Saturn) [inline main.py]: I am getting a `TypeError: object of type 'generator' has no len()`. Changing the generator comprehension to a list comprehension seemed to fix this for me. And it does print the bottom 3 in red on 18 and 20 team leagues which is an improvement.

**Comment by Saturn** [comment]
Also this season the French league will only be relegating the bottom 2 teams. The output should probably reflect that also.

**Comment by thurask** [comment]
Maybe a list of special cases to check?

Something like

```
elif league.CLEAGUE <= team["position"] <= league.RELEGATED
```

defined for each lague?

**Comment by Saturn** [comment]
It might be a good idea to have some kind of list of properties for each league.
* Number of teams
* CL spots
* EL spots
* Number of teams relegated

This could simplify things when it comes to weird cases like French league only having two relegated teams. And every few years the UEFA coefficients can affect the number of CL spots etc. For example Italy could potentially overtake England and get an extra spot.

**Comment by architv** [comment]
@thurask @Saturn I have merged the branch for now. I'll be adding other properties based on  #11

**Comment by architv** [comment]
@Saturn @thurask I've added property for each league for relegation and champions league spots. Would add EL spots too.

**Pull request merged**